src/eval/lang.rs

```diff
@@ -137,6 +162,28 @@ impl<F: LurkField, C: Coprocessor<F>> Lang<F, C> {
     }
 }
 
+pub struct Binding<F: LurkField, C: Coprocessor<F>> {
```

Comment:
A docstring on top of this structure would help convey intended meangning. Circuit-less coprocessor isn’t something which usage and purpose is immediate and intuitive (at least to me).

Reply:
I did add a doc string in follow-up, but `Binding` doesn't relate to circuitless coprocessors. An example of utility will be supporting meta-Lurk (e.g. Clutch's `!(prove…)` and `!(verify …)` as 'normal' Lurk functions implemented for the evaluator but with no provable equivalent.